### PR TITLE
[bsblan] Minor SAT fixes

### PR DESCRIPTION
**Pull request by lsiepel**
Signed-off-by: lsiepel <leosiepel@gmail.com>

Minor refactoring:
- add null annotation
- checkstyle

### CODE REVIEW

**Comment by lsiepel** [comment]
@jlaur i guess the build failed is not related to this PR? 
Seems something wrong with:  osgi.wiring.package=org.eclipse.californium.elements.util)(version>=2.7.0)(!(version>=3.0.0)))"]

File: bundles/org.openhab.binding.bsblan/src/main/java/org/openhab/binding/bsblan/internal/api/BsbLanApiCaller.java [hunk 1]
```diff
@@ -17,10 +17,10 @@
 import java.io.ByteArrayInputStream;
 import java.io.IOException;
 import java.io.InputStream;
-import java.nio.charset.Charset;
 import java.util.Set;
 import java.util.stream.Collectors;
 
+import org.eclipse.californium.elements.util.StandardCharsets;
```
Comment (lolodomo) [inline bundles/org.openhab.binding.bsblan/src/main/java/org/openhab/binding/bsblan/internal/api/BsbLanApiCaller.java:23]: Why this strange import? 
You can get UTV 8 standard charger from common Java class.

**Review by lolodomo — CHANGES_REQUESTED** [review]

File: bundles/org.openhab.binding.bsblan/src/main/java/org/openhab/binding/bsblan/internal/api/BsbLanApiCaller.java (hunk 1 above)
Comment (lsiepel) [reply bundles/org.openhab.binding.bsblan/src/main/java/org/openhab/binding/bsblan/internal/api/BsbLanApiCaller.java]: That is where it should have come from.

**Review by lolodomo — APPROVED** [review]
LGTM

**Comment by lolodomo** [comment]
Waiting for a GO to merge.

**Comment by lsiepel** [comment]
> Waiting for a GO to merge.

I also wanted to remove the dependency on org.apache.commons. but i have not yet seen an follow-up to the question in that issue about core/comon string util classes.

**Comment by lsiepel** [comment]
> Waiting for a GO to merge.

Thunderbirds GO :-)

**Comment by jlaur** [comment]
I believe this will conflict with #14202.

**Comment by lsiepel** [comment]
> I believe this will conflict with #14202.

I think it is the other way around. ;-) 
Anyway, there is no real conflict as both PR's make the exact same change to the same two lines. Just merge one and the other might have to adapt. Not sure how git handles such case, let's see.

**Comment by jlaur** [comment]
> I think it is the other way around. ;-)

To be clear, I meant they both conflict with each other. 🙂

> Anyway, there is no real conflict as both PR's make the exact same change to the same two lines. Just merge one and the other might have to adapt. Not sure how git handles such case, let's see.

I didn't compare in that detail, but you are right - if the changes are exactly the same, there should be no conflict. We'll see.

**Review by jlaur — APPROVED** [review]
Thanks!

**Pull request merged**